nixos/modules/system/boot/zram-as-tmp.nix

```
@@ -0,0 +1,96 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.boot.tmp;
+in
+{
+  options = {
+    boot.tmp = with lib; {
+      useZram = mkOption {
+        type = types.bool;
```

```suggestion
    boot.tmp = {
      useZram = lib.mkOption {
        type = lib.types.bool;
```
Or inherit the relevant functions at the top of the file.